### ISSUE TITLE
[github] git-bug bridge pull incomplete comments wrong status

### ISSUE THREAD

**Issue by frodeaa** [MichaelMure/git-bug#385]
There seems to be an bug in how the github bridge pull issues. Some issues gets wrong status and incomplete comment history

Tested the bridge on this repo and got wrong number of issues open, 75 vs 73. 

```
git bug bridge pull
git bug ls -s open | wc -l
75
```

**Comment by frodeaa** [comment]
Notices also that there are issues with wrong title

```
 f09dbd9   open    feat: Link to code from comme… ◼ ◼  Tanguy ⧓ Herrm…     5 💬    4 minutes ago
 fd71207   open    feat: Link to code from comme… ◼ ◼  Michael Muré (…    10 💬    3 weeks ago
```

The `fd71207` has wrong title, it should be `Reflections on the change needed in the data model` https://github.com/MichaelMure/git-bug/issues/212

**Comment by frodeaa** [comment]
https://github.com/MichaelMure/git-bug/issues/167 is an issue that the wrong status of open

**Comment by rng-dynamics** [comment]
Fixed by #534

**Comment by frodeaa** [comment]
Nice!

```
git bug ls status:open | wc -l
88
git bug ls status:closed | wc -l
128
```
